pkg/localkube/storage_provisioner.go

```diff
@@ -123,7 +125,7 @@ func StartStorageProvisioner(lk LocalkubeServer) func() error {
 
 	// Create an InClusterConfig and use it to create a client for the controller
 	// to use to communicate with Kubernetes
-	config := rest.Config{Host: "http://localhost:8080"}
+	config := rest.Config{Host: net.JoinHostPort("localhost", strconv.Itoa(lk.APIServerInsecurePort))}
```

Comment:
It appears to work without the http:// but I will add it back.  net.JoinHostPort does not do this automatically